piet/src/text.rs

```diff
@@ -28,7 +29,59 @@ pub trait TextLayoutBuilder {
     fn build(self) -> Result<Self::Out, Error>;
 }
 
+/// # Text Layout
+///
+/// A text position corresponds roughly to unicode code units. Because piet uses rust utf-8
+/// strings, this means that text position corresponods roughly with byte index.
```

Comment:
"corresponds"

Also I think we can sharpen up the language. "Text position is defined in utf-8 code units, as is standard for Rust strings."